Tasks/HelmDeployV0/Tests/L0.ts

```diff
@@ -1,16 +1,194 @@
+        delete process.env[shared.TestEnvVars.updatedependency];
+        delete process.env[shared.isHelmV3];
+        delete process.env[shared.TestEnvVars.releaseName];
+        delete process.env[shared.TestEnvVars.waitForExecution];
+        delete process.env[shared.TestEnvVars.arguments];
+        delete process.env[shared.TestEnvVars.chartName];
+        delete process.env[shared.TestEnvVars.chartPath];
+        delete process.env[shared.TestEnvVars.connectionType];
+        delete process.env[shared.TestEnvVars.command];
+        delete process.env[shared.TestEnvVars.chartType];
+    });
+
+    after((done) => {
+        done();
+    });
+
+    it('Run successfully with Helm install (version 3) with chart name', function(done: MochaDone) {        
+        const tp = path.join(__dirname, 'TestSetup.js');
+        const tr: ttm.MockTestRunner = new ttm.MockTestRunner(tp);
+        process.env[shared.TestEnvVars.connectionType] = shared.ConnectionTypes.KubernetesServiceConnection;
+        process.env[shared.TestEnvVars.command] = shared.Commands.install;
+        process.env[shared.TestEnvVars.chartType] = shared.ChartTypes.Name;
+        process.env[shared.TestEnvVars.chartName] = shared.testChartName;
+        process.env[shared.TestEnvVars.releaseName] = shared.testReleaseName;
+        process.env[shared.TestEnvVars.failOnStderr] = 'ture';
+        process.env[shared.isHelmV3] = 'true';
+
+        tr.run();
+        assert(tr.stdout.indexOf("v3") != -1, 'Helm version 3 installed');
+        assert(tr.stdout.indexOf('STATUS: deployed') != -1, `Release created with NAME: ${shared.testReleaseName}`);
+        assert(tr.stdout.indexOf('# Source: testChartName/templates/serviceaccount.yaml') != -1, `Manifests extracted from release ${shared.testReleaseName}`);
+        assert(tr.stdout.indexOf('DeploymentDetailsApiResponse: {"mockKey":"mockValue"}') != -1, 'Web response for pushing metadata to evidence store');
+        assert(tr.succeeded, 'task should have succeeded');
+        done();
     });
 
-    after(() => {
+    it('Run successfully with Helm install (version 2) with chart name', function(done: MochaDone) {        
+        const tp = path.join(__dirname, 'TestSetup.js');
+        const tr: ttm.MockTestRunner = new ttm.MockTestRunner(tp);
+        process.env[shared.TestEnvVars.connectionType] = shared.ConnectionTypes.KubernetesServiceConnection;
+        process.env[shared.TestEnvVars.command] = shared.Commands.install;
+        process.env[shared.TestEnvVars.chartType] = shared.ChartTypes.Name;
+        process.env[shared.TestEnvVars.chartName] = shared.testChartName;
+        process.env[shared.TestEnvVars.releaseName] = shared.testReleaseName;
+        process.env[shared.TestEnvVars.failOnStderr] = 'false';
+        process.env[shared.TestEnvVars.namespace] = shared.testNamespace;
+
+        tr.run();
+        assert(tr.stdout.indexOf("v2") != -1, 'Helm version 2 installed');
```

Comment:
NIT: Shouldn't the message say `Helm v2 should have been installed`?

Reply:
Yes. I'll update.